{{ cookiecutter.package_name }}/pyproject.toml

```diff
@@ -54,10 +54,14 @@ zip-safe = false
 include-package-data = true
 
 [tool.setuptools.packages.find]
+include = ["{{ cookiecutter.module_name }}*"]
```

Comment:
I checked this entire block rendered correctly for the three test cases.

It has the Nabil approved amount of line breaks.